traffic_ops/app/lib/API/Configs/ApacheTrafficServer.pm

```diff
@@ -1451,24 +1449,51 @@ sub logging_dot_config {
 			$text .= "	Format = '" . $format . " '\n";
 			$text .= "}\n";
 		}
+	}
+
+	for ( my $i = 0; $i < $max_log_objects; $i = $i + 1) {
+		my $log_filter_field = "LogFilter";
+		if ( $i > 0 ) {
+			$log_filter_field = $log_filter_field . "$i";
+		}
+		my $log_filter_name = $data->{$log_filter_field . ".Name"} || "";
+		if ( length($log_filter_name) > 0) {
+			my $filter = $data->{$log_filter_field . ".Filter"};
+			$filter =~ s/"/\\\"/g;
```

Comment:
Does this work? I think this escaping might be incorrect. It's used to escape the format string, because that's supposed to be an actual string. (I believe the escaping is actually unnecessary for double quotes, though, because the value is single quoted in the construction.)

But here, we expect something like `accept("field MATCH value")` and escaping the quotes is wrong.

Reply:
The filter is defined in single quotes just like the Format string: filter.accept('%<{X-ATS-SVC-ID}cqh> CONTAIN logds') is what I've been testing with.  If lua will accept the unescaped double quote for something like filter.accept(field CONTAIN ") then the escaping is indeed unnecessary (and I can't imagine the use case for searching for quotes in a value anyway)

Reply:
After looking at it I think it's confusing because as written the filter MUST be defined as "accept('something MATCH/CONTAIN something') or "reject('something MATCH/CONTAIN something') and there's no validation of that format.   Perhaps different profile parameters for each type (AcceptFilter and RejectFilter) so the single quotes can be added in the code?